notebooks/retriever-reader-pipelines/prompt_reader_LLM.ipynb

```diff
@@ -9,8 +9,8 @@
      "name": "stdout",
      "output_type": "stream",
```

Comment:
Please add comments describing what the notebook does

Reply:
done